app/src/main/res/values/colors.xml

```diff
@@ -40,5 +40,5 @@
     <color name="dark_mode_elevation_1">#24272e</color>
     <color name="dark_mode_semi_transparent">#e624272e</color>
     <color name="translucent_tool_bar">#730A0D14</color>
-    <color name="warning">#E84646</color>
+    <color name="warning">#aa0045</color>
```

Comment:
If we got this color from the color system, lets make sure that it has the same name as in the color system. This will ensure that we speak the same language when talking to our designers